src/users/usersReducers.js

```diff
@@ -20,10 +13,10 @@ import { NULL_ARRAY } from '../nullObjects';
 
 const initialState: UsersState = NULL_ARRAY;
 
-const initUsers = (state: UsersState, action: InitUsersAction): UsersState => action.users;
-
-const realmInit = (state: UsersState, action: RealmInitAction): UsersState =>
-  action.data.realm_users;
+const realmInit = (state: UsersState, action: RealmInitAction): UsersState => {
+  console.log('action.data', action.data);
```

Comment:
I think you didn't intend to check this line in. :wink:

Reply:
Huh, yeah!  Eslint gives warnings when we use `console.log`. I think it should also cause the CI to fail (which it currently doesn't). Opened #2820 for this.